clusterloader2/testing/storage/pod-startup/ephemeral-volumes/config.yaml

```diff
@@ -0,0 +1,97 @@
+#Constants
+{{$NODES_PER_NAMESPACE := DefaultParam .NODES_PER_NAMESPACE 1}}
+{{$PODS_PER_NODE := DefaultParam .PODS_PER_NODE 1}}
+{{$LOAD_TEST_THROUGHPUT := DefaultParam .LOAD_TEST_THROUGHPUT 10}}
+
+{{$VOLUMES_PER_POD := 4}}
+{{$VOLUME_TYPE := DefaultParam .VOLUME_TYPE "EmptyDir"}}
+{{$PROVISION_VOLUME := DefaultParam .PROVISION_VOLUME false}}
+
+{{$APP_NAME := "pod-load"}}
+{{$GROUP := "pod-with-ephemeral-volume-startup-latency"}}
+
+#Variables
+# TODO think about this!
```

Comment:
Ownerless/issue-less todo, please fix

Reply:
It's not clear in comments but I added this and other TODOs because I need feedbacks about these parameters and their values.

Reply:
See my comment above.
I actually believe that we need two separate tests:
1. first focus only on the first two cases from the KEP (which by design are running on a very small cluster)
2. separately address the third case, which I think has to be written a bit differently

Assuming we're focuing on (1) here, I would vote for simplifying this case and not think about namespaces, etc. at all here. I would be much easier to understand then,

Reply:
I'll delete the parameters that I need for the (2) tests